### PR TITLE
feat(x1): add fitness score in calibration.launch and delete default value

### PR DESCRIPTION
**Pull request by yn-mrse**
## Description

Since fitness score is a parameter that can vary depending on the map,
change requests occur at different timings than software function updates.
Therefore, it seems desirable to manage fitness scores separately from this repository.

In this PR, the default value of the fitness score parameter value has been deleted, and allow arguments to be specified when starting calibration.launch.

With this change, fitness_score_threshold must be specified as an additional option when starting a command in mode:=map_based.

The startup command is as follows.

ros2 launch extrinsic_calibration_manager calibration.launch.xml map_path:=/opt/autoware/maps/ pointcloud_map_file:=0.05_map0_converted_calib_removed_ground.pcd lanelet2_map_file:=lanelet2_map.osm map_calibration_area_file:=0.05_map0_cutout_calib_area_map.pcd mode:=map_based sensor_ model:=aip_x1 vehicle_model:=ymc_golfcart_m0 **fitness_score_threshold:=0.001**

Note that when starting in a mode other than map_based, there is no need to use the fitness_score_threshold option.
Therefore, in this case, it is necessary to specify a specification that will work without setting the fitness_score_threshold option.

## Related links

[TIER IV INTERNAL LINK, management ticket](https://tier4.atlassian.net/browse/AEAP-730)

### When executing the launch command with mode:=map_based in calibration.launch
1. Preventing startup due to unintended parameter values
   - Avoid unintentionally starting with different parameter values when fitness_score_threshold option is not specified.
2. Successful startup/normal operation based on expected parameter values
   - When fitness_score_threshold option is specified, the parameter value is applied and it starts normally.

#### 1. Preventing startup due to unintended parameter values
The command below intentionally does not specify the fitness_score_threshold option, but it has been confirmed that if this is executed, calibration_launch can be prevented from starting.
````
ros2 launch extrinsic_calibration_manager calibration.launch.xml map_path:=/opt/autoware/maps/ pointcloud_map_file:=0.05_map0_converted_calib_removed_ground.pcd lanelet2_map_file:=lanelet2_map.osm map_calibration_area_file:=0.05_map0_cutout_calib_area_map.pcd mode:=map_based sensor_ model:=aip_x1 vehicle_model:=ymc_golfcart_m0
````

#### 2. Successful startup/normal operation based on expected parameter values
The command below is a normal command with the fitness_score_threshold option specified, it was confirmed that when this was executed, calibration_launch could be started normally and calibration could be executed normally.

````
ros2 launch extrinsic_calibration_manager calibration.launch.xml map_path:=/opt/autoware/maps/ pointcloud_map_file:=0.05_map0_converted_calib_removed_ground.pcd lanelet2_map_file:=lanelet2_map.osm map_calibration_area_file:=0.05_map0_cutout_calib_area_map.pcd mode:=map_based sensor_ model:=aip_x1 vehicle_model:=ymc_golfcart_m0 fitness_score_threshold:=0.001
````

### When executing the launch command in a mode other than mode:=map_based in calibration.launch
3. Prevention of inhibition of startup due to fitness score threshold value
   - Startup should not fail even if fitness_score_threshold option is not specified.

#### 3. Prevention of inhibition of startup due to fitness score threshold value

Confirmed operation under the condition of mode:=ground_plane. We confirmed that launch was successful and continued operation without adding the fitness_score_threshold option.

````
ros2 launch extrinsic_calibration_manager calibration.launch.xml map_path:=/opt/autoware/maps/ pointcloud_map_file:=0.05_map0_converted_calib_removed_ground.pcd lanelet2_map_file:=lanelet2_map.osm map_calibration_area_file:=0.05_map0_cutout_calib_area_map.pcd mode:=ground_place sensor_ model:=aip_x1 vehicle_model:=ymc_golfcart_m0
````

## Notes for reviewers

- Although we have confirmed the activation of aip_x1_1, we have not yet confirmed its operation using the corresponding rosbag.
- Since aip_xx1 is not applicable, operation has not been confirmed. Please note that if you do not use the fitness_score_threshold option, it may not work properly.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml [hunk 1]
```diff
@@ -6,6 +6,8 @@
   <arg name="map_calibration_area_file" default="map_calibration_area.pcd"/>
   <arg name="use_calibration_area_map" default="true" />
   <arg name="calibration_rviz" default="true"/>
+  <arg name="fitness_score_threshold" />
```
Comment (h-ohta) [inline sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]: The aip_xx1 is outside of our project. Do we have to fix it too?

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based_sensor_kit.launch.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <let name="input_map_without_wall_topic_name" value="/map/pointcloud_map_without_wall" />
   <arg name="use_calibration_area_map" default="true" />
   <arg name="accumulate_frame_num" default="2" />
-  <arg name="fitness_score_threshold" default="0.001" />
+  <arg name="fitness_score_threshold" />
```
Comment (yn-mrse) [inline sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based_sensor_kit.launch.xml]:
```suggestion
  <arg name="fitness_score_threshold" default="0.001" />
```

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based_sensors.launch.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
   <let name="input_map_with_wall_topic_name" value="/map/pointcloud_map" />
   <let name="input_map_without_wall_topic_name" value="/map/pointcloud_map_without_wall" />
   <arg name="accumulate_frame_num" default="2" />
-  <arg name="fitness_score_threshold" default="0.001" />
+  <arg name="fitness_score_threshold" />
```
Comment (yn-mrse) [inline sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based_sensors.launch.xml]:
```suggestion
  <arg name="fitness_score_threshold" default="0.001" />
```

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml (hunk 1 above)
Comment (yn-mrse) [reply sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]: I reached [agreement](https://star4.slack.com/archives/C04CKSQ7ZFZ/p1696230427088419?thread_ts=1695974107.937419&cid=C04CKSQ7ZFZ) with our project to apply the changes only to aip_x1 and aip_x1_1, not aip_xx1.
Therefore I will revert the aip_xx1 changes.

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml [hunk 4]
```diff
@@ -6,6 +6,8 @@
...
+
```
Comment (yn-mrse) [inline sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]:
```suggestion
```

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml (hunk 1 above)
Comment (yn-mrse) [reply sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]: I fixed at https://github.com/tier4/CalibrationTools/pull/129/commits/8db247c45692f903fb13a255e03a150dadfcd1e5 , https://github.com/tier4/CalibrationTools/pull/129/commits/12865fa8e8e88eca7c5a5086120ee1c84ad474c2 , and https://github.com/tier4/CalibrationTools/pull/129/commits/373c039ba61b3026c4bd52c89887100a2822bd0f .

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml [hunk 5]
```diff
@@ -17,6 +17,7 @@
     <include file="$(find-pkg-share extrinsic_calibration_manager)/launch/$(var sensor_model)/map_based_sensor_kit.launch.xml">
       <arg name="vehicle_id" value="$(var vehicle_id)" />
       <arg name="use_calibration_area_map" value="$(var use_calibration_area_map)" />
+      <arg name="fitness_score_threshold" value="$(var fitness_score_threshold)" />
```
Comment (h-ohta) [inline sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]: Do you revert this change too?

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml [hunk 6]
```diff
@@ -25,6 +26,7 @@
     <include file="$(find-pkg-share extrinsic_calibration_manager)/launch/$(var sensor_model)/map_based_sensors.launch.xml">
       <arg name="vehicle_id" value="$(var vehicle_id)" />
       <arg name="use_calibration_area_map" value="$(var use_calibration_area_map)" />
+      <arg name="fitness_score_threshold" value="$(var fitness_score_threshold)" />
```
Comment (h-ohta) [inline sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]: Do you revert this change too?

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml (hunk 5 above)
Comment (yn-mrse) [reply sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]:
```suggestion
```

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml (hunk 6 above)
Comment (yn-mrse) [inline sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]:
```suggestion
```

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml (hunk 5 above)
Comment (yn-mrse) [reply sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]: very sorry. There was unfinished business.
I will correct this.

File: sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml (hunk 6 above)
Comment (yn-mrse) [reply sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml]: very sorry. There was unfinished business.
I will correct this.

**Review by h-ohta — APPROVED** [review]

**Comment by yn-mrse** [comment]
Related linksのリンク先が誤っていることに気がついたため、正式なリンクへ変更しました。
